common/kvstore/src/test/java/org/apache/spark/util/kvstore/RocksDBIteratorSuite.java

```diff
@@ -45,4 +51,37 @@ protected KVStore createStore() throws Exception {
     return db;
   }
 
+  @Test
```

Comment:
I think it should also be feasible to put it into `RocksDBSuite`, and there's no need to redefine `CustomType1`.